docs/source/gettingstarted.rst

```diff
@@ -62,99 +55,121 @@ All other inputs follow those of the ``Variable`` class.
-To declare an objective function, assign a Posynomial (or Monomial) to a variable name, such as ``objective``.
-
-.. code-block:: python
-
-    objective = 1/(x*y*z)
-
-By convention, the objective is the function to be *minimized*. If you wish to *maximize* a function, take its reciprocal. For example, the code above creates an objective which, when minimized, will maximize ``x*y*z``.
-
+    type(c)  # gpkit.nomials.PosynomialInequality
 
 Formulating a Model
 ================
 
-The ``Model`` class represents an optimization problem. To create one, pass an objective and list of Constraints:
+The ``Model`` class represents an optimization problem. To create one, pass an objective and list of Constraints.
+
+By convention, the objective is the function to be *minimized*. If you wish to *maximize* a function, take its reciprocal. For example, the code below creates an objective which, when minimized, will maximize ``x*y*z``.
 
 .. code-block:: python
 
     objective = 1/(x*y*z)
     constraints = [2*x*y + 2*x*z + 2*y*z <= S,
                    x >= 2*y]
-    gp = Model(objective, constraints)
+    m = Model(objective, constraints)
 
 
 Solving the Model
 ==============
 
+When solving the model you can change the level of information that gets printed to the screen with the ``verbosity`` setting. A verbosity of 1 (the default) prints warnings and the solution; a verbosity of 2 prints solve time, a verbosity of 3 prints solver output, and a verbosity of 0 prints nothing.
+
 .. code-block:: python
 
-    sol = gp.solve()
+    sol = m.solve(verbosity=0)
 
 
 Printing Results
 ================
 
+Here we can manually print the solution table, with the same result as if the verbosity argument had been left blank above.
+
 .. code-block:: python
 
     print sol.table()
 
+::
+
+    Cost
+    ----
+     15.59 [1/m**3]
+
+    Free Variables
+    --------------
+    x : 0.5774  [m]
+    y : 0.2887  [m]
+    z : 0.3849  [m]
+
+    Constants
+    ---------
+    S : 1  [m**2]
+
+    Sensitivities
+    -------------
+    S : -1.5
+
 .. code-block:: python
 
     print "The x dimension is %s." % (sol(x))
 
+::
+
+    The x dimension is 0.577351209028 meter.
+
 Sensitivities and dual variables
 ================================
 
-When a GP is solved, the solver returns not just the optimal value for the problem’s variables (known as the "primal solution") but also, as a side effect of the solving process, the effect that scaling the less-than side of each constraint would have on the overall objective (called the "dual solution", "shadow prices", or "posynomial sensitivities").
+When a GP is solved, the solver returns not just the optimal value for the problem’s variables (known as the "primal solution") but also, as a side effect of the solving process, the effect that scaling the :math:`\leq 1` of each canonical constraint would have on the overall objective (called the "dual solution", "shadow prices", or "posynomial sensitivities").

```

Comment:
not sure these are called shadow prices in a GP context - I've only heard that for LPs